conbench/app/_plots.py

```diff
@@ -413,59 +440,79 @@ def gen_js_callback_click_on_glyph_show_run_details(repo_string):
     )
 
 
-def time_series_plot(history, benchmark, run, height=380, width=1100):
-    # log.info("Time series plot for:\n%s", json.dumps(history, indent=2))
+def time_series_plot(
+    samples: List[HistorySample], benchmark, run, height=380, width=1100
+):
+    # log.info("Time series plot for:\n%s", json.dumps(samples, indent=2))
 
-    unit = history[0]["unit"]
-    with_dist = [h for h in history if h["distribution_mean"]]
-    formatted, axis_unit = _should_format(history, unit)
+    unit = samples[0].unit
+    with_dist = [s for s in samples if s.zscorestats.rolling_mean]
```

Comment:
I like that `h["distribution_mean"]` now is `s.zscorestats.rolling_mean`. As said before, we can totally change the name of `zscorestats` or the overall structure, but "distribution mean" was lacking context, which is why I was confused by it for weeks. The context really is a specific analysis method, and the existence of this method should be reflected in code.

Reply:
Love this, yes. I'll think some more on zscorestats and if there's a better name (something like distribution_distance_stats maybe? zscore is nice in that it doesn't use a relatively common word like distribution, but what this is fundamentally about is taking a distribution of points and then seeing if the current thing is close to that distribution or far from that distribution). But regardless, this is much much clearer as is IMO

Reply:
> I'll think some more on zscorestats and if there's a better name

Thanks!